routes/company-routes.js

```diff
@@ -234,6 +234,30 @@ module.exports = function (app) {
         }
     })
 
+    app.post('/getTripData', async (req, res) => {
+        const contactEmail = req.body.contactEmail;
+        const licensePlate = req.body.licensePlate;
+        const cO2Consumed = req.body.cO2Consumed;
```

Comment:
same deal here, we could accept a dictionary instead of having to make separate variables for all the properties

Reply:
yuppers